### PR TITLE
[script.kodi.windows.update] 1.0.9

### PR DESCRIPTION
**Pull request by Lunatixz**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.kodi.windows.update/addon.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <source>https://github.com/Lunatixz/KODI_Addons/tree/master/script.kodi.windows.update</source>
```
Comment (enen92) [inline script.kodi.windows.update/addon.xml]: spdx if possible

File: script.kodi.windows.update/service.py [hunk 2]
```diff
@@ -38,38 +39,52 @@ def log(msg, level=xbmc.LOGDEBUG):
...
-    def getBuild(self): 
+        
+    def getPlatform(self): 
+        log('getPlatform')
         count = 0
-        build = ''
         try:
-            while not xbmc.Monitor().abortRequested() and count < 3:
-                count += 1
+            while not xbmc.Monitor().abortRequested() and count < 15:
```
Comment (enen92) [inline script.kodi.windows.update/service.py]: reuse monitor?

Comment (Lunatixz) [reply script.kodi.windows.update/service.py]: It's a lazy method, probably could have written better; with a single Monitor call.

File: script.kodi.windows.update/addon.xml (hunk 1 above)
Comment (Lunatixz) [reply script.kodi.windows.update/addon.xml]: Are we using direct .git paths now?

Comment (enen92) [reply script.kodi.windows.update/addon.xml]: it was the closest line I could find to the `<license>` element :) nothing wrong with the repo url

**Comment by Lunatixz** [comment]
@enen92 should be good to go

**Pull request closed without merging**